src/app/containers/OnDemandHeading/StyledTvHeadingContainer.jsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import OnDemandHeading from '.';
+
+const StyledTvHeadingContainer = styled(OnDemandHeading)`
+  @media screen {
```

Comment:
Not sure what the `@media screen` query is doing here & in `StyledRadioHeadingContainer.jsx` when it only declares a media type?

Reply:
Apparently overriding headline styles? https://github.com/bbc/simorgh/commit/36227c65b8c267e13451e9331f00bf96abe1e2c2

I'll play around with it

Reply:
Ah that's right, I think it was to override the default padding set by `@bbc/psammead-headings` - `Headline` https://github.com/bbc/psammead/blob/b339f0167cce21ccd8bed382622e6797627aee24/packages/components/psammead-headings/src/index.jsx#L20

It was to avoid using `!important` - but it's still a bit narsty.

Ideally this stuff probably wouldn't have defaults set by our open-source component library (that anyone using it then has to overwrite), but implementing that would be a beast of a task.

See if you can find a different way but if not we might have to live with it.